src/mrack/data/provisioning-config.yaml

```diff
@@ -95,6 +95,17 @@ beaker:  # beaker provider specific values
     distros:
         # list of beaker distros key is used in metadata distro is used in generated jobxml
         fedora-32: Fedora-32%
+        c9s: Centos-Stream-9%
```

Comment:
I reckon it should be `CentOS-Stream-9%` as on line `107`

Reply:
I will fix that

Reply:
Fixed